static/app/views/performance/transactionDetails/finishSetupAlert.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from '@emotion/styled';
+
+import Button from 'app/components/button';
+import {IconLightning} from 'app/icons';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization, Project} from 'app/types';
+import {trackAdvancedAnalyticsEvent} from 'app/utils/advancedAnalytics';
+
+export default function FinishSetupAlert({
+  organization,
+  project,
+}: {
+  organization: Organization;
+  project: Project;
+}) {
+  return (
+    <AlertBar>
+      <IconLightning />
+      <TextWrapper>
+        {t(
+          'You are viewing a sample transaction. Configure performance to start viewing real transactions.'
+        )}
+      </TextWrapper>
+      <Button
+        size="xsmall"
+        priority="primary"
+        target="_blank"
```

Comment:
Can you wrap this with `<ExternalLink>` here? If not, we should add `rel: 'noreferrer noopener'` although probably not a big deal since it's to docs.

Reply:
@billyvg I can turn the button into an `ExternalLink` by adding the `external` prop so that should be good enough